examples/kmeans.rs

```diff
@@ -33,7 +34,7 @@ fn main() {
     let dataset = DatasetBase::from(data);
     let rng = thread_rng(); // Random number generator
     let n_clusters = 3;
-    let model = KMeans::params_with_rng(n_clusters, rng)
+    let model = KMeans::params_with(n_clusters, rng, LInfDist)
```

Comment:
Shouldn't this be L2 distance for Euclidean distance?

Reply:
Yes, you're right; fixed!